payload-manifests/featuregates/featureGate-SelfManagedHA-Default.yaml

```diff
@@ -136,6 +136,9 @@
                     {
                         "name": "ProcMountType"
                     },
+                    {
+                        "name": "ResilientWatchCacheInitialization"
```

Comment:
@JoelSpeed will this FG be available after this PR is included in a promoted image?

Reply:
yes it was, i have just checked the feature crd in a must-gather and `ResilientWatchCacheInitialization` was disabled. Thanks!